staging/src/k8s.io/apiserver/pkg/server/filters/timeout_test.go

```diff
@@ -258,9 +258,11 @@ func TestErrConnKilled(t *testing.T) {
 		t.Fatal("expected to receive an error")
 	}
 
-	// we should only get one line for this, not the big stack from before
+	// we expect to see one line for this, not the big stack from before,
+	// and to address a flake we check for at most two lines (the executing
+	// handler of a timed out request from a different test prints to the log)
 	capturedOutput := readStdErr()
-	if strings.Count(capturedOutput, "\n") != 1 {
+	if strings.Count(capturedOutput, "\n") > 2 {
 		t.Errorf("unexpected output captured actual = %v", capturedOutput)
```

Comment:
@p0lyn0mial had discovered a flake - inner handler (of a request that had been timed out) from a different test can write to the log and this will fail the test. (my change adds a single log entry from such an inner handler)

Maybe there is a different way we can check this?

Reply:
> inner handler (of a request that had been timed out) from a different test can write to the log and this will fail the test

Have you identified that test? This test assumes the other tests are run in serial.